maggy/maggy/services/inbox.py

```
@@ -0,0 +1,201 @@
+            client = anthropic.AsyncAnthropic(api_key=self.cfg.ai.api_key)
+            msg = await client.messages.create(
+                model=self.cfg.ai.model,
+                max_tokens=2000,
+                messages=[{"role": "user", "content": prompt}],
+            )
+            text = msg.content[0].text if msg.content else ""
+            # Extract JSON
+            start = text.find("{")
+            end = text.rfind("}")
+            data = json.loads(text[start:end + 1]) if start >= 0 else {"rankings": []}
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)